index.html

```diff
@@ -340,5 +340,32 @@ <h1>
         </div>
       </div>
     </section>
+    <div class="form-div">
+      <form action="https://formspree.io/f/xbjboedq" method="POST" class="form">
+        <h2>Contact me</h2>
+
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input
+          type="text"
+          maxlength="30"
+          placeholder="Enter your name here"
+          required
+        />
+        <input type="email" placeholder="Enter your email here" required />
+        <textarea
+          name="text-area"
+          id="text-area"
+          cols="20"
+          rows="7"
+          maxlength="500"
+          placeholder="Write your message here"
+        ></textarea>
+        <button type="submit">Get In Touch</button>
```

Comment:
- [ Optional ]  As it stated in project requirement I recommend you to Implement  Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
  
![Capture](https://user-images.githubusercontent.com/93598189/192628277-25adbfa4-ce81-48f3-8a28-e0d9130d7478.PNG)

Reply:
Fixed this one aswell.